technical-specs/001-Identity-MVP-Spec/001-Identity-Spec.md

```diff
@@ -0,0 +1,125 @@
+
+# App Center Identity Spec 
+
+This spec details the App Center Identity service that we will launch a public preview MVP.
+
+## Overview 
+
+### TL;DR 
+The goal is to deliver an identity management service in App Center that allows app developers to: 
+
+1. Create and manage an Azure AD B2C resource/tenant in App Center 
+2. Configure the Azure AD B2C resource/tenant for the following identity approaches in order of priority: Google, Facebook, Email/Password, Microsoft 
+3. View and search for user identities, and the ability to edit in Azure 
+4. Use the provided identity to integrate with other services in App Center: 
+   - Crashes and analytics events (i.e. Users of the app will be tagged to their crash/analytics data) 
+   - Push notifications (i.e. App developers can send push notifications to specific users)
+   - Data Storage/GDPR 
+
+## User scenarios 
+
+### Overview 
+
+We are prioritizing the following platforms and providers: 
+- Platforms: Android, iOS, Xamarin, React Native
```

Comment:
Can you highlight some of the background for this order? Based on customer demand, our list is similar, but also includes a few more cross-platform JS frameworks like Ionic and Cordova. Also, any consideration for Flutter? We haven't had huge demand, but appears to be a growing favorite.

Reply:
Thanks @danieldobalian for your comment! Got it on your request to support AD B2E accounts.

The order is:
1. Native platforms: Android, iOS
2. Xamarin
3. RN

For 2 and 3, we may also change this based on user's feedback post the launch of App Center Identity. So keep giving us feedback here!

Reply:
Order is based on usage in App Center. Our focus has mostly been on mobile platforms so adoption is the largest there. React Native, Xamarin and Cordova are really wrappers for our native SDK's so it makes sense to tackle Android and iOS first before working on Xamarin and React Native improvements.